goodkey/good_key.go

```diff
@@ -89,6 +97,20 @@ func (policy *KeyPolicy) GoodKey(key crypto.PublicKey) error {
 			return berrors.BadPublicKeyError("public key is forbidden")
 		}
 	}
+	if policy.dbCheck != nil {
+		digest, err := core.KeyDigest(key)
+		if err != nil {
+			return err
+		}
+		// TODO: probably should add a ctx to GoodKey
```

Comment:
I think we should resolve this TODO as part of this PR prior to merge. Also, `context.TODO()` is appropriate for these occasions as an additional reminder.